src/Randomizer.Data/Services/TrackerStateService.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Randomizer.Data.WorldData;
+using Randomizer.Data.WorldData.Regions;
+using Randomizer.Shared.Models;
+
+namespace Randomizer.Data.Services
+{
+    public class TrackerStateService : ITrackerStateService
+    {
+        private readonly RandomizerContext _randomizerContext;
+        private readonly ILogger<TrackerStateService> _logger;
+
+        public TrackerStateService(RandomizerContext dbContext, ILogger<TrackerStateService> logger)
+        {
+            _randomizerContext = dbContext;
+            _logger = logger;
+        }
+
+        public void CreateState(World world, GeneratedRom generatedRom)
```

Comment:
Considering you're doing database stuff here, these methods should really be async